### DISCUSSION TITLE
Code Documentation

### DISCUSSION THREAD

**Discussion by ali-alhuwswe** [mixcore/mix.core#330] [Q&A] [answered]
Hello guys i just fount mixcore cms , what a great cms i was just started working in idea close to mixcore but i am glad i found it but could any one please tell me where i can find code denomination i did not find any thing in the  official website

**Comment by Smilefounder** [comment, marked as answer]
Hi @ali-alhuwswe thanks for your interesting in Mixcore CMS. FYI, we are working on the documentation at the moment as well as cleaning the source code to make it cleaner after the several of an experiment.

The documents that we are working now existing on the repos below:
- https://api-docs.mixcore.org/api/index.html (https://github.com/mixcore/docs.api)
- http://mixcore.dev (https://github.com/mixcore/docs)

I will keep everyone posted on the progress for the documentation soon. 

Best regards,
Mixcore team

**Comment by Smilefounder** [comment]
Hi @ali-alhuwswe You can see a quick dev environment setup guide here https://docs.mixcore.org/dev-getting-started that I just updated.

Fyi, https://github.com/mixcore/mix.theme.material-kit this is an 80% completed theme that shows you all of the basic code to use with the CMS.
